hawkular-alerts-bus/src/main/java/org/hawkular/alerts/bus/listener/AvailDataListener.java

```diff
@@ -115,7 +115,7 @@ protected void onBasicMessage(AvailDataMessage msg) {
     protected String convertReceivedMessageClassNameToDesiredMessageClassName(String className) {
 
         if (className.equals("org.hawkular.metrics.component.publish.AvailDataMessage")) {
-            return "org.hawkular.alerts.bus.messages.AvailDataMessage";
+            return "org.hawkular.alerts.bus.api.AvailDataMessage";
```

Comment:
You mean to change the return "org.hawkular.alerts.bus.api.AvailDataMessage" per return AvailDataMessage.getClass().getCanonicalName() ?
If so, I don't see major differences. I think maintaining the constant can be better readable.